openlibrary/templates/work_search.html

```diff
@@ -76,6 +76,16 @@ <h1>$_("Search")</h1>
         <input type="radio" name="mode" value="printdisabled" id="mode-search-printdisabled" class="search-mode">
         <label for="mode-search-printdisabled">$_('Print Disabled')</label>
       </span>
+      $if ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians')):
```

Comment:
Instead of removing the checkbox, let's keep this around to pass down to the SearchWork card and relax to librarians instead of super so all necessary folks see it

Reply:
Done